easycv/utils/checkpoint.py

```diff
@@ -33,7 +39,18 @@ def load_checkpoint(model,
     Returns:
         dict or OrderedDict: The loaded checkpoint.
     """
-    if not filename.startswith('oss://'):
+    if load_style == 'timm':
+        print('use timm load_pretrained')
+        if filename.endswith('.npz'):
+            return model.backbone.model.load_pretrained(filename)
+        else:
+            backbone_module = importlib.import_module(
+                model.backbone.model.__module__)
+            return load_pretrained(
+                model.backbone.model,
+                filter_fn=backbone_module.checkpoint_filter_fn if hasattr(
+                    backbone_module, 'checkpoint_filter_fn') else None)
```

Comment:
these logic should not be placed here, but in classification model, and we should decide whether to use timm style using cfg rather than adding other argument